composer.json

```diff
@@ -55,14 +55,14 @@
         "symfony/options-resolver": ">=5.4 <6",
         "symfony/psr-http-message-bridge": "^2",
         "symfony/routing": ">=5.4 <6",
-        "thenetworg/oauth2-azure": "^1.4",
+        "thenetworg/oauth2-azure": "^2.0",
```

Comment:
I understand this is the main reason for your contribution. As this is a major upgrade, we will have to test it thoroughly to make sure, everything still works. So please give us some time here 😉